common/include/pcl/common/impl/transforms.hpp

```diff
@@ -306,6 +305,52 @@ transformPointCloud (const pcl::PointCloud<PointT> &cloud_in,
 }
 
 
+void
+transformPointCloud(pcl::PointCloud<pcl::PointXY>& cloud_in, 
```

Comment:
`cloud_in` should be const

Reply:
I get this error if I make it const:
```
 error: passing ‘const pcl::PointXY’ as ‘this’ argument discards qualifiers [-fpermissive]
  345 |         Eigen::Vector2f pf = cloud_in[i].getVector2fMap();

```

Reply:
You need to add the const overloads as well (basically right now getVector2fMap says that the data can be changed. You need to give an overload to the same function that promises not to change data on being called)

Reply:
I have another getVector2fMapConst() function. so should I put it in a try except?